assets/scripts/app/MiscHTMLStuff.jsx

```diff
@@ -6,13 +6,158 @@
  * @module MiscHTMLStuff
  */
 import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import SkyBackground from './SkyBackground'
+import ScrollIndicators from './ScrollIndicators'
+import { infoBubble } from '../info_bubble/info_bubble'
+import { animate } from '../util/helpers'
+import { MAX_CUSTOM_STREET_WIDTH } from '../streets/width'
+import { app } from '../preinit/app_settings'
+
+class MiscHTMLStuff extends React.Component {
```

Comment:
I can see why a lot of this component needed to be expanded to deal with the interactions between different child components here. As a result I think we should probably just call this something else instead of `MiscHTMLStuff` since it's no longer miscellaneous and has a pretty specific goal. Should we rename this component to `Street.jsx` (or `StreetView` or `StreetSection`)?